pkg/projects/Microsoft.NETCore.DotNetHost/Microsoft.NETCore.DotNetHostResolver.pkgproj

```diff
@@ -16,6 +16,9 @@
     <Dependency Include="Microsoft.NETCore.DotNetHost">
       <Version>$(HostFullVersion)</Version>
     </Dependency>
+    <Dependency Include="Microsoft.NETCore.DotNetAppHost">
```

Comment:
Does this mean that when transitive closure of DotNetHostResolver is published, both dotnet.exe and apphostexe will be published as well? I would expect that we will remove the dependency on DotnetHost as part of this change since that is only used in SharedFW scenarios and thus, never published.